apiary.apib

```diff
@@ -216,9 +297,9 @@ or set account properties or destroy their accounts.
 
 Destroying an account, and changing account properties
 like username and password, require to pass the user's
-password as `x-password` header.
+bearer token as `Authorization` header.
 
-### Sign Up [PUT /session/account?include=session]
+### Sign Up [PUT /session/account]
```

Comment:
As discussed with @gr2m, `PUT /session/account` will just sign up the user without creating a session. In the future we might decide to have an additional endpoint to do both sign up and sign in in the same request.